package.json

```diff
@@ -4,8 +4,13 @@
   "description": "User interface for the HashiCorp Nomad scheduler.",
   "repository": "https://github.com/iverberk/nomad-ui",
   "main": "src/main.js",
+  "bin": {
+    "webpack": ".bin/webpack",
```

Comment:
I only have a node_modules/.bin directory locally. Is that what you want? The npm build command currently doesn't do anything, just exits right away with error code 0.